tests/verifyDump_test.js

```diff
@@ -5,6 +5,24 @@ const {I,
 
 Feature('Tests for Dump Tool');
 
+const host_volume=process.cwd() + '/tests/output/sftp/';
+const sftp = {
+    username: 'foo',
+    password: 'password',
+    directory: '/upload/',
+};
+
+BeforeSuite(async ({ I, codeceptjsConfig }) => {
+    await I.verifyCommand(`mkdir -p ${host_volume}`);
```

Comment:
re: the chmod, yes i had to do this because the Host volume settings are not portable, since they are host-dependent and might not work on any other machine. I did try with docker host user details "foo" or the group but still it didnt work.
I have left the folder same for uploads as well instead of tmp as we decided let the downloads be in /output/sftp earlier